pipeline_tools/pipelines/smartseq2/smartseq2.py

```diff
@@ -1,5 +1,21 @@
 from pipeline_tools.shared import metadata_utils
 from pipeline_tools.shared.http_requests import HttpRequests
+from pipeline_tools.shared.reference_id import ReferenceId
+
+
+REFERENCES = {
+    ReferenceId.Human.value: {
+        'hisat2_ref_trans_name': 'gencode_v27_trans_rsem',
+        'rrna_intervals': 'gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/gencode.v27.rRNA.interval_list',
+        'hisat2_ref_index': 'gs://hca-dcp-mint-test-data/reference/HISAT2/genome_snp_tran.tar.gz',
+        'genome_ref_fasta': 'gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/GRCh38.primary_assembly.genome.fa',
+        'hisat2_ref_trans_index': 'gs://hca-dcp-mint-test-data/reference/HISAT2/gencode_v27_trans_rsem.tar.gz',
+        'rsem_ref_index': 'gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/gencode_v27_primary.tar',
+        'gene_ref_flat': 'gs://hca-dcp-mint-test-data/reference/GRCh38_Gencode/GRCh38_gencode.v27.refFlat.txt',
+        'hisat2_ref_name': 'genome_snp_tran',
+        'stranded': 'NONE',
```

Comment:
Can you explain why "stranded" is something that depends on the reference? I'm not entirely sure what it's used for

Reply:
@kishorikonwar Charley mentioned you might know

Reply:
Ping @kishorikonwar